share/csm_share/shr/shr_orb_mod.F90

```diff
@@ -78,8 +78,10 @@ real(SHR_KIND_R8) pure FUNCTION shr_orb_cosz(jday,lat,lon,declin,dt_avg)
       shr_orb_cosz =  shr_orb_avg_cosz(jday, lat, lon, declin, dt_avg)
    else
       shr_orb_cosz = sin(lat)*sin(declin) - &
-           cos(lat)*cos(declin) * &
-           cos((jday-floor(jday))*2.0_SHR_KIND_R8*pi + lon)
+      &              cos(lat)*cos(declin)*cos(jday*2.0_SHR_KIND_R8*pi + lon)
```

Comment:
We need to have the current CIME routine in here and not the one brought in from ACME. As in PJR comment's it seems that the newer version (i.e. the one in the current CIME) should have been brought into AME in a separate PR. Not sure how you want to proceed but there should not be a change in shr_orb_mod.F90 in this PR.